PartyIcons/View/NameplateView.cs

```diff
@@ -178,14 +241,15 @@ public void SetupForPC(XivApi.SafeNamePlateObject npObject)
                     break;
 
                 case NameplateMode.SmallJobIcon:
-                    iconID = GetClassIcon(npObject.NamePlateInfo);
+                    name = GetStateNametext(_configuration.ShowPlayerStatus ? iconID : -1, _iconPrefix, SeStringUtils.SeStringFromPtr(name));
```

Comment:
Perhaps configuration check should be moved inside those methods, since currently its duplicated way too many times.